content/en/docs/features/encrypted-images.md

```diff
@@ -120,19 +149,93 @@ spec:
       labels:
         app: nginx
       annotations:
+        io.katacontainers.config.hypervisor.kernel_params: "agent.guest_components_rest_api=all agent.aa_kbc_params=cc_kbc::${KBS_ADDRESS}"
```

Comment:
The `rest_api` settings are mostly orthogonal to encrypted images. Maybe we should leave that out. Also setting it to all is not secure by default.

Reply:
OK - I'll remved part of param and leave only cc_kbs related part